redpanda/templates/tests/test-api-status.yaml

```diff
@@ -15,7 +15,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */}}
 {{- /* TODO test fails if SASL is enabled */}}
```

Comment:
Should this line be deleted after your changes to enable running SASL without TLS (non-blocker)?